circle.yml

```diff
@@ -0,0 +1,31 @@
+---
+version: 2
+jobs:
+  build:
+    docker:
+      - image: library/docker:17
+    working_directory: /tmp/build
+    environment:
+      DOCKER_USERNAME: travisciresin
+      DOCKER_IMAGE: balena/open-balena-api
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: |
+          docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
```

Comment:
We rely here on `set -x` not being in effect and CircleCI not printing the expanded env var which feels somewhat risky as we could end up with the password printed in clear text in the build logs if anything changes.

Reply:
Good point - this is taken direct from `resin-api` and I don't think we're actually push/pulling any private images - so maybe it can go @dfunckt ?

Reply:
Ah indeed, that's a good point -- yes, login can be completely removed.

Reply:
Wait, I think it's CircleCI that pushes the images to DockerHub, not ResinCI, so it can't be removed.